### ISSUE TITLE
#enhancement Re-read /some/ previous journal entries when "Start Monitor" is clicked.

### ISSUE THREAD

**Issue by fredjk-gh** [Xjph/ObservatoryCore#5]
Capturing an idea I threw out on your last stream.

If Observatory core were to re-read entirety of the current journal file and/or the previous journal file (if entry into the current system was not found in the current one) to try retrieve the details for the most recent system in order to restore context from where we left off. (Another way of putting it, re-read the latest two journal files, keeping records for only latest system encountered in a buffer, then prime the grid display with those journal records before monitoring for new records in real-time.)

Motivation 1:

- Explore, FSS/DSS a body, consider it interesting but you're {tired|time for dinner|etc}, land on it. Log off {close game|close Observatory}.
- Later, resume game, start Observatory. However the body/signals context for this body is now gone and Observatory doesn't now where we are, or how many signals the body has.

Motivation 2:

- I forgot to click "Start Monitor" before playing, jumped a system, FSS'd and/or mapped some things, then realized I forgot.

With this:
- Explorer would initialize the display the other interesting things in the system I'm currently in (ie. potentially cues for my next stop in the system).
- Botanist would know the biological signals count and the name of the body we're parked on, so when the first ScanOrganic event comes in, it's not "blank | 0 | Organic name" in the grid.
- Its far faster than re-reading all before starting.
- It's automatic, and a much more consistent experience.
